examples/hf_serialization.py

```diff
@@ -242,37 +325,36 @@ def hf_main(args):
     logger.info("GPU: " + utils.get_gpu_name())
     logger.info("PYTHON USED RAM: " + utils.get_mem_usage())
 
-    serialize_model(model, model_config, output_prefix)
-
-    tokenizer = AutoTokenizer.from_pretrained(args.input_directory)
```

Comment:
Why did we remove this? We do want to copy the tokenizer data, and this also updates the tokenizer to what format   is supported.

Reply:
@harubaru mentioned this not being necessary in our discussion in the aforementioned thread, so it was the first thing I removed and hadn't revisited it since.

Reply:
Tokenizer doesn't seem necessary for just serializing the weights to object storage unless we can upload the tokenizer onto object storage. If the HF ID is preserved then the tokenizer can just be fetched that way

Reply:
Tokenizers are now being saved in #76